### ISSUE TITLE
Spec pom.xml needs updates for html generation

### ISSUE THREAD

**Issue by kwsutter** [eclipse/microprofile-opentracing#59]
The spec pom.xml needs some updates to properly generate the html.  Currently, the html generation does not properly display the version of the spec.  The pdf generation is good, we just need to update the html generation.

Here's the content of the config spec pom.xml (and the html generation looks good):

```
                    <execution>
                        <id>output-html</id>
                        <phase>generate-resources</phase>
                        <goals>
                            <goal>process-asciidoc</goal>
                        </goals>
                        <configuration>
                            <backend>html5</backend>
                            <attributes>
                                <revnumber>${project.version}</revnumber>
                                <revremark>${revremark}</revremark>
                                <revdate>${revisiondate}</revdate>
                            </attributes>
                        </configuration>
                    </execution>
```
Once we get this figured out, then we need to update the download site with the properly generated 1.0 html.

This was discussed on our gitter thread:
https://gitter.im/eclipse/microprofile-opentracing

**Comment by kwsutter** [comment]
I propose that we clean up the pom.xml for the next release, but don't bother with creating a respin for 1.0.1 or 1.0a just to update the html version of the spec.

**Comment by kwsutter** [comment]
Resolved via PR #61.
